app/org/sagebionetworks/bridge/services/email/MimeTypeEmail.java

```diff
@@ -28,8 +33,12 @@
         checkArgument(messageParts != null && !messageParts.isEmpty());
         
         this.subject = subject;
-        this.senderAddress = senderAddress;
-        this.recipientAddresses = recipientAddresses;
+        this.senderAddress = escapeEmailAddress(senderAddress);
+        this.recipientAddresses = Lists.transform(recipientAddresses, new Function<String,String>() {
+            @Override public String apply(String address) {
+                return escapeEmailAddress(address);
+            }
```

Comment:
This anonymous class will be recreated every time someone calls this constructor. It would be cleaner to have a static class-level inner class (anonymous or named).